libraries/stdlib/samples/test/samples/text/strings.kt

```diff
@@ -79,4 +79,10 @@ class Strings {
         assertPrints(byteArray.toString(charset), "Hello")
     }
 
+    @Sample
+    fun stringToUpperCase() {
+        assertPrints("Hello".toUpperCase(), "HELLO")
+        assertPrints("Hello".toUpperCase(Locale.ENGLISH), "HELLO")
```

Comment:
I wonder, is there such an example that shows why locale could be used for toUpper/LowerCase? I mean so that it would result in a different result compared to no locale overload.

Reply:
@ilya-g 
Thank you for review.
I understood that the following is unnecessary.
After excluding the following points,I will request a review again.
> assertPrints("Hello".toUpperCase(Locale.ENGLISH), "HELLO")